.travis.yml

```diff
@@ -13,13 +13,17 @@ python:
 install:
   - pip install codecov
   - pip install .[dev]
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install -U black; fi
 script:
   - export PYTHONPATH=$PYTHONPATH:$(pwd)
   # * E501 - line length limit
   # * E123 - indentation on data structures
   # * W504 - line break after binary operator
-  - flake8 . --ignore=F403,E501,E123,E128,W504 --exclude=docs,build
+  - flake8 . --ignore=F403,E501,E123,E128,W504,W503 --exclude=docs,build
```

Comment:
Why was this added in particular?

Reply:
The answer is more difficult than I thought, but the short version is: W503 is actually incompatible with pep8. Flake8 actually ignores W503 by default, but when you customize the ignore list it is re-enabled. By the way, with flake8's default settings the black formatted code would only trigger E501. But since we "need" to ignore that one we also have to ignore W503. That would actually already be enough at least for now, since black makes sure F403, E123, E128 and W504 are not triggered.
Or we could go with blacks own flake8 ignore list:
`flake8 . --ignore=E203,E266,E501,W503`

What do you think?

Reply:
The only one that I feel truly passionate about is E501, which is in both lists. 
I could go either way on this. 

If we stay with the current list then we have to add that and continue to support it. It is also not as clean cut as just using black. 

If we move to black's list, then we might have to do some clean up. I think all of it would be magic with Black and easy clean up except for F403. 

I am leaning towards black's list, but could be talked into the other. Someone would need to make sure that the F403 error was cleaned up. 

The other option is to manually force blacks list with the addition of F403 & then put F403 as a TODO item that is in a separate PR/Ticket. 

So, I punt back to you. How do you want to move forward?

Reply:
Also, let me save you the trouble... https://www.flake8rules.com/